clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/converters/UpsertEntityTagsAtomicOperationConverter.java

```diff
@@ -59,14 +59,20 @@ public UpsertEntityTagsAtomicOperationConverter(ObjectMapper objectMapper,
   }
 
   public AtomicOperation convertOperation(Map input) {
-    return new UpsertEntityTagsAtomicOperation(
-      retrySupport, front50Service, accountCredentialsProvider, entityTagsProvider, this.convertDescription(input)
-    );
+    return buildOperation(convertDescription(input));
+  }
+
+  public AtomicOperation buildOperation(UpsertEntityTagsDescription description) {
+    description.getTags().forEach(UpsertEntityTagsAtomicOperationConverter::setTagValueType);
+    return new UpsertEntityTagsAtomicOperation(retrySupport,
```

Comment:
minor -- line breaks look off here too

```
return new UpsertEntityTagsAtomicOperation(
  retrySupport,
  front50service,
  ...
);
```

has been the pattern I've tried to follow for at least this area of the codebase